git/git.go

```diff
@@ -64,6 +64,10 @@ func ResolveRef(ref string) (*Ref, error) {
 		return nil, err
 	}
 	lines := strings.Split(outp, "\n")
+	if len(lines) <= 1 {
+		err := errors.New("git can't resolve ref : " + ref)
```

Comment:
I prefer using `fmt.Errorf` simple error messages that build dynamic strings:

```go
err := fmt.Errorf("git can't resolve ref: %q", ref)
```

This is nitpicky, and not worth waiting on you to fix. Just some Go feedback :)